kubernetes/crd/dataset.json

```diff
@@ -0,0 +1,35 @@
+{
+	"apiVersion": "apiextensions.k8s.io/v1",
+	"kind": "CustomResourceDefinition",
+	"metadata": { "name": "datasets.new.artificialwisdom.cloud" },
+	"spec": {
+		"group": "new.artificialwisdom.cloud",
+		"versions": [{
+			"name": "v1",
+			"served": true,
+			"storage": true,
+			"schema": {
+				"openAPIV3Schema": {
+					"type": "object",
+					"properties": {
+						"spec": {
+							"type": "object",
+							"properties": {
+								"source": { "type": "string" },
```

Comment:
is source our source, or the authorative source of the dataset?

If so, you may consider adding mimetyping, or some other marker, if there is something more reasonable then mimetypes, which at my last look in 2000 were far from. ;=-)

Reply:
In accordance with the spirit of #108, the `source` specifies a path on HF Hub where a dataset can be loaded, if we pass that path directly to HF Datasets. I'll add descriptions here.

It would be nice, longer-term, to have information about how the datasets are stored in HF Hub, but unfortunately their structure involves custom loader scripts which are written in Python and stored on the Hub. It's an inherently unsafe and undocumented way of working with data.